setup.cfg

```diff
@@ -28,7 +28,7 @@ python_requires = >=3.7
 zip_safe = False
 install_requires =
     Babel>=2.8
-    cookiecutter>=1.7.1,<1.8.0
```

Comment:
As far as I can see from PyPI, `1.7.3` _is_ [the latest release for the 1.x series](https://cookiecutter.readthedocs.io/en/stable/HISTORY.html) :sweat_smile: 

Have you tried your luck by bumping it to e.g. `<2.2`?
The breaking changes don't sound like they'd be affecting `invenio-cli` in a negative way.

Reply:
Oh, my bad... I managed to do a wrong 1 line PR, maybe I should have checked the version before remembering some number :sweat_smile:  

This also resolved the cached package issue in the CI, apparently everything looks fine :) I also was not expecting any problems.